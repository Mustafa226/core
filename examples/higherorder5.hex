% test higher order creating predicate input to external atom
seta(u).
seta(v).
seta(w).

pred(setb).

P(X) :- pred(P), seta(X), X != v.

setc(X) :- &testSetMinus[seta,setb](X).
