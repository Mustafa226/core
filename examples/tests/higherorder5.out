{setb(u),setb(w),setc(v)}
